.github/workflows/release.yaml

```diff
@@ -25,9 +26,13 @@ jobs:
           ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
           ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.ARTIFACT_SIGNING_PRIVATE_KEY }}
 
-      - uses: ffurrer2/extract-release-notes@v2
-        id: release_notes
-
-      - uses: ncipollo/release-action@v1
+      - name: Extract release notes
+        id: extract-release-notes
+        uses: ffurrer2/extract-release-notes@v2
         with:
-          body: ${{ steps.release_notes.outputs.release_notes }}
+          release_notes_file: RELEASE_NOTES.md
```

Comment:
This is not the name of our file. We can rely on the default

Reply:
It's used by `--notes-file` option, due to https://github.com/cashapp/licensee/pull/357#issue-2506630710.

Reply:
Oh this _writes_ the file, not reads it.